examples/Tools/ChooseRadioAccessTechnology/ChooseRadioAccessTechnology.ino

```diff
@@ -0,0 +1,126 @@
+
+/*
+  Radio Access Technology selection for Arduino MKR GSM 1400
+  This sketch allows you to select your preferred Radio Access
+  Technology (RAT).
+  You can choose among 2G, 3G(UMTS) or a combination of both.
+  Selecting JUST ONE technology will speed up the modem's network
+  registration A LOT!
+  The chosen configuration will be saved to modem's internal memory
+  and will be preserved through MKR GSM 1400 sketch uploads.
+  In order to change the RAT, you will need to run this sketch again.
+  Circuit:
+  - MKR GSM 1400 board
+  - Antenna
+  - SIM card
+  Created 19 August 2019
+  by Riccardo Rizzo
+*/
+
+#include <MKRGSM.h>
+
+GSM gms(true);
+void setup() {
+  Serial.begin(921600);
+  while (!Serial);
+
+  MODEM.begin();
+  while (!MODEM.noop());
+
+  for (int i = 0; i < 80; i++) Serial.print("*");
+  Serial.println();
+  Serial.println("This sketch allows you to select your preferred");
+  Serial.println("GSM Radio Access Technology (RAT).");
+  Serial.println();
+  Serial.println("You can choose among 2G, UMTS(3G) or a combination of both.");
+  Serial.println();
+  Serial.println("The chosen configuration will be saved to modem's internal memory");
+  Serial.println("and will be preserved through MKR GSM 1400 sketch uploads.");
+  Serial.println();
+  Serial.println("In order to change the RAT, you will need to run this sketch again.");
+  for (int i = 0; i < 80; i++) Serial.print("*");
+
+  Serial.println();
+  Serial.println();
+  Serial.println("Please choose your Radio Access Technology:");
+  Serial.println();
+  Serial.println("    0 - 2G only");
+  Serial.println("    1 - 3G only");
+  Serial.println("    2 - 2G preferred, 3G as failover (default)");
```

Comment:
From the AT command reference module the default is UART= 1 = GSM / UMTS (dual mode)